tests/EventListener/DeserializeListenerTest.php

```diff
@@ -61,6 +61,24 @@ public function testDoNotCallWhenRequestNotManaged()
         $listener->onKernelRequest($eventProphecy->reveal());
     }
 
+    public function testDoNotCallWhenRequestRequestIsFalse()
```

Comment:
It's only *request* processing that's being skipped, not all processing. In a similar fashion we could say "skip *response* processing".